yajsapi/executor/index.ts

```diff
@@ -800,7 +833,13 @@ export class Executor implements ComputationHistory {
       }
       emit(new events.ComputationFinished());
     } catch (error) {
-      logger.error(`fail= ${error}`);
+      if (error instanceof AsyncGeneratorBreak) {
+        work_queue.close();
+        done_queue.close();
+        logger.info("User decided to stop getting new results.");
```

Comment:
I'm not sure I like this message. Why mention "User" and imply any "decision"? The computation is stopped simply by a `break` statement in code.